yourtrading-ai/fishnet-cod
# Introduce full ENV configurability

Using Pydantic's `BaseSettings` class. Every property of this class is settable with environment variables or with a `.env` file.

**Note:** All properties need to be prefixed with `FISHNET_`.

## How to test
Set `FISHNET_TEST_CHANNEL` to `True` and check if at startup the API outputs something like `Using channel: FISHNET_TEST_2023-04-24 15:22:01.0124512`
